examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_pb.bal

```diff
@@ -1,86 +1,65 @@
-// This is an auto generated client stub which is used to connect to gRPC server.
 import ballerina/grpc;
 import ballerina/io;
 
-// Non-blocking client.
-public type ChatStub object {
-    public grpc:Client clientEndpoint;
-    public grpc:Stub stub;
+public type ChatClient client object {
```

Comment:
Shall we add more comments to this example?

Reply:
Fixed in 0980f96